<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientWindow</class>
 <widget class="QMainWindow" name="PatientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PatientWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>-10</y>
      <width>771</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="baseSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(14, 10, 25);
color: rgb(232, 223, 224);</string>
    </property>
    <property name="text">
     <string>Patients</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="verticalFrame_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>220</width>
      <height>541</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>400</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>220</width>
      <height>600</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(14, 10, 25);</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="Tabs">
       <item>
        <widget class="QPushButton" name="HButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>25</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>200</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(71, 23, 246);
color: rgb(232, 223, 224);</string>
         </property>
         <property name="text">
          <string>Home</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>25</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>200</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(71, 23, 246);
color: rgb(232, 223, 224);</string>
         </property>
         <property name="text">
          <string>Patients</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="AButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>25</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>200</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(71, 23, 246);
color: rgb(232, 223, 224);</string>
         </property>
         <property name="text">
          <string>Appointments</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="SButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>25</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(71, 23, 246);
color: rgb(232, 223, 224);</string>
       </property>
       <property name="text">
        <string>Appointments</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="search">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>470</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Patients</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="OHIP">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>470</y>
      <width>351</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>450</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter OHIP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Patient Data:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Injection History:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addPatient">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>New Patient</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addInjection">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>180</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Injection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addAppoint">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>New Appointment</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Appointments:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editPatient">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>180</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Patient</string>
    </property>
   </widget>
   <widget class="QListWidget" name="appoint">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>321</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="editAppoint">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>360</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Appointment</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>360</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addFile">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>360</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>New File</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>220</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Files:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openFile">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>360</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>360</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QListWidget" name="files">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>240</y>
      <width>321</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="shots">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>60</y>
      <width>321</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="demo">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>321</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
